draft-ietf-quic-transport.md

```diff
@@ -2578,8 +2606,11 @@ so.
 To support this process, a token is sent by endpoints.  The token is carried in
 the NEW_CONNECTION_ID frame sent by either peer, and servers can specify the
 stateless_reset_token transport parameter during the handshake (clients cannot
-because their transport parameters don't have confidentiality protection).  This
-value is protected by encryption, so only client and server know this value.
+because their transport parameters don't have confidentiality protection).
+Tokens sent via NEW_CONNECTION_ID frames are invalidated when their associated
+connection ID is retired via a CONNECTION_ID_FINISHED frame
+({{frame-connection-id-finished}}).  This value is protected by encryption, so
+only client and server know this value.
```

Comment:
With the intervening discussion of tokens and frames, it's no longer obvious what "This value" is.  I'd either swap the last two sentences or explicitly say "The token" again.  Probably with a preference toward the sentence-swap, since the invalidation of tokens is unrelated to them being issued under encryption.

Reply:
Sounds good, I'll swap them